app/controllers/state_file/questions/ny_third_party_designee_controller.rb

```diff
@@ -0,0 +1,9 @@
+module StateFile
+  module Questions
+    class NyThirdPartyDesigneeController < AuthenticatedQuestionsController
```

Comment:
Since this data is on the review page, you should add `include ReturnToReviewConcern` so that if someone edits this from the review page they go back to review after submitting.